src/typechecker/FStar.TypeChecker.Rel.fs

```diff
@@ -4196,18 +4238,32 @@ let resolve_implicits' env is_tac g =
           end
         end
   in
+
+  (*
```

Comment:
cc @nikswamy , before starting fixpoint, iterate over the `solution_types` compartment.